src/webapi/document.rs

```diff
@@ -67,4 +68,16 @@ impl Document {
             ).into_reference_unchecked()
         }
     }
+
+    /// Returns the <body> or <frameset> node of the current document, or null if no such element exists.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/body)
+    // https://html.spec.whatwg.org/#dom-document-body
```

Comment:
We try to link to the IDL in the specs, so this link should be: `https://html.spec.whatwg.org/#the-document-object:dom-document-body`